.github/workflows/deploy-prep.yml

```
@@ -0,0 +1,739 @@
+              targetPort: $([ "${service}" = "api" ] && echo "8080" || echo "3000")
+            type: ClusterIP
+          EOF
+            fi
+          done
+
+      - name: Generate environment-specific kustomization
+        env:
+          ENVIRONMENT: ${{ needs.prepare-deployment.outputs.environment }}
+        run: |
+          mkdir -p "k8s/overlays/${ENVIRONMENT}"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)